AmplifyPlugins/Core/AWSPluginsCore/Model/Support/QueryPredicate+GraphQL.swift

```diff
@@ -60,30 +104,50 @@ extension QueryPredicate {
 }
 
 extension QueryPredicateOperation: GraphQLFilterConvertible {
-    var graphQLFilter: GraphQLFilter {
-        return [field: [self.operator.graphQLOperator: self.operator.value]]
+    func graphQLFilterInternal(_ modelSchema: ModelSchema? = nil) -> GraphQLFilter {
+        let filterValue = [self.operator.graphQLOperator: self.operator.value]
+        guard let modelSchema = modelSchema else {
+            return [field: filterValue]
+        }
+        return [columnName(modelSchema, forField: field): filterValue]
+    }
+
+    func columnName(_ modelSchema: ModelSchema, forField fieldName: String) -> String {
+        guard let modelField = modelSchema.field(withName: fieldName) else {
+            return field
```

Comment:
code coverage for when you pass in a `field` that does not resolve to a ModelField